java/src/jmri/jmrix/can/cbus/CbusConfigurationManager.java

```diff
@@ -308,6 +314,29 @@ public CbusCabSignalManager getCabSignalManager() {
         return cabSignalManager;
     }
 
+    protected ConsistManager consistManager = null;
+
+    public ConsistManager getConsistManager() {
+        if ( adapterMemo.getDisabled() ) {
+            return null;
+        }
+        if (consistManager == null) {
+            if (adapterMemo.getProgModeSwitch().equals(ProgModeSwitch.EITHER)) {
+                // Could be either programmer or command station
+                if (getProgrammerManager().isAddressedModePossible()) {
+                    // We have a command station
+                    consistManager = new NmraConsistManager(get(jmri.CommandStation.class));
+                } else {
+                    consistManager = null;
+                }
+            } else {
+                // Command station is always avaliable
```

Comment:
Can you explain this comment? Command station is always available when not i programmer mode??

Reply:
if the earlier (adapterMemo.getProgModeSwitch().equals(ProgModeSwitch.EITHER)) is false then it means the hardware supports both programmer and command station at the same time. The Pi-SPROG 3 Plus has two track connections and always provides a command station (i.e. with ops mode) on one of them.

Reply:
I see.

I suggest changing the line 324 from
`if (adapterMemo.getProgModeSwitch().equals(ProgModeSwitch.EITHER)) {`
to
`if (adapterMemo.getProgModeSwitch() == ProgModeSwitch.EITHER) {`

It would make the code easier to read and easier to understand since `ProgModeSwitch` is an enum.